fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1697,6 +1697,11 @@ columnAliases
 partitionNames
     : TEMPORARY? (PARTITION | PARTITIONS) '(' identifier (',' identifier)* ')'
     | TEMPORARY? (PARTITION | PARTITIONS) identifier
+    | lakePartition
+    ;
+
+lakePartition
```

Comment:
to be honest, I didn't find a better name.  Many partition related name has been used, But these existed exprs don't meet my requirement.